From b9eb8be1fc5344f20ef70690c9c27cd2f107d13b Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Wed, 20 Apr 2022 16:34:05 +0300
Subject: [PATCH 382/471] ARM: dts: at91: sama7g5ek: remove DMA from i2c nodes

DMA transfers do not work on I2C when FIFO is enabled. For now, disable
DMA transfers for I2C.

Fixes: 7540629e2fc7 ("ARM: dts: at91: add sama7g5 SoC DT and sama7g5-ek")
Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Reviewed-by: Tudor Ambarus <tudor.ambarus@microchip.com>
---
 arch/arm/boot/dts/at91-sama7g5ek.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/at91-sama7g5ek.dts b/arch/arm/boot/dts/at91-sama7g5ek.dts
index e0bc3ea6095d..9ac1781bb619 100644
--- a/arch/arm/boot/dts/at91-sama7g5ek.dts
+++ b/arch/arm/boot/dts/at91-sama7g5ek.dts
@@ -247,6 +247,7 @@ &flx1 {
 	status = "okay";
 
 	i2c1: i2c@600 {
+		dmas = <0>, <0>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_i2c1_default>;
 		i2c-analog-filter;
@@ -413,6 +414,7 @@ &flx8 {
 	status = "okay";
 
 	i2c8: i2c@600 {
+		dmas = <0>, <0>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_i2c8_default>;
 		i2c-analog-filter;
@@ -455,6 +457,7 @@ &flx9 {
 	status = "okay";
 
 	i2c9: i2c@600 {
+		dmas = <0>, <0>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_i2c9_default>;
 		i2c-analog-filter;
-- 
2.34.1

